specification/storage/resource-manager/readme.md

```diff
@@ -38,6 +38,7 @@ input-file:
 - Microsoft.Storage/stable/2019-06-01/storage.json
 - Microsoft.Storage/stable/2019-06-01/blob.json
 - Microsoft.Storage/stable/2019-06-01/file.json
+- Microsoft.Storage/stable/2019-06-01/privatelinks.json
```

Comment:
Is that really necessary? 
I don't see any rest API defined in this file.
This file only contains some definations which is referenced by storage.json.